### DISCUSSION TITLE
no UDP support, or I am missing something?

### DISCUSSION THREAD

**Discussion by XBeg9** [yggdrasil-network/yggdrasil-go#1002] [Q&A] [answered]
Hi, I see only tcp, tls schemas, no UDP support?

**Comment by neilalexander** [comment, marked as answer]
There is no UDP support. The current iteration of the Yggdrasil protocol currently relies on reliable stream ordering to avoid routing loops, so we can’t support UDP at this point. 

This may change in v0.5 as the protocol is being reworked.
